Tools/clinic/clinic.py

```diff
@@ -67,19 +69,26 @@ def __repr__(self):
 
 sig_end_marker = '--'
 
+Appender = Callable[[str], None]
+Outputter = Callable[[None], str]
 
-_text_accumulator_nt = collections.namedtuple("_text_accumulator", "text append output")
+class _TextAccumulator(NamedTuple):
+    text: list[str]
+    append: Appender
+    output: Outputter
 
 def _text_accumulator():
     text = []
     def output():
         s = ''.join(text)
         text.clear()
         return s
-    return _text_accumulator_nt(text, text.append, output)
+    return _TextAccumulator(text, text.append, output)
 
 
-text_accumulator_nt = collections.namedtuple("text_accumulator", "text append")
+class TextAccumulator(NamedTuple):
```

Comment:
It is very confusing: `TextAccumulator` and `_TextAccumulator`. Maybe we can add some suffix to one of them?

Reply:
@erlend-aasland wrote this little bit, so I'll let him comment on what the best name might be!

Reply:
These are confusing names indeed (FWIW, I simply used the old names without the `_nt` suffix). We can exploit the opportunity to name them better; or we can just do that in a follow-up refactor PR (I kinda lean toward the latter).